### PR TITLE
fix(WelcomeScreen): Fix welcome screen scroll view

### PR DESCRIPTION
**Pull request by jamesbrooks94**
# Description

This PR fixes the button being outside the scrollable view and unavailable on US, this also affects UK and Sweden

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [X] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots
Before:
image

After:
image

## Out of scope and potential follow-up

Unit tests should be added

### CODE REVIEW

**Comment by abubuwe** [comment]
The country selector flag in the top-right corner now appears to be unclickable from your screenshot, as it is behind the notification bar?

**Comment by jamesbrooks94** [comment]
@abubuwe I had just scrolled down, it is still clickable :)

File: src/features/register/Welcome1Screen.tsx [hunk 1]
```diff
@@ -1,139 +1,140 @@
...
+  const userService = new UserService();
 
-                    <TouchableOpacity style={styles.countryFlag} onPress={() => this.props.navigation.navigate('CountrySelect', {patientId: null})}>
-                        <Image style={styles.flagIcon} source={this.flagIcon()}/>
-                    </TouchableOpacity>
+  useEffect(() => {
+    userService.getUserCount().then((response) => {
+      if (response) {
+        const userCount = parseInt(response, 10);
+        console.log(userCount);
```
Comment (selvach) [inline src/features/register/Welcome1Screen.tsx:49]: Could you remove this debug code, please

**Comment by abubuwe** [comment]
Just tried on a smaller iOS device (iPhone 5s & 8) in emulator and also work for me

**Comment by jamesbrooks94** [comment]
> Just tried on a smaller iOS device (iPhone 5s & 8) in emulator and also work for me

I don’t have permissions to merge

**Pull request merged**